reference/errors/error_codes.md

```diff
@@ -132,6 +132,10 @@ The requested task status is invalid. Please use one of the [possible values](/r
 
 The requested task type is invalid. Please use one of the [possible values](/reference/api/tasks.md#type).
 
+## `invalid_task_canceled_by_filter`
+
+The provided query parameter is invalid. Please use one of the [available query parameters](/reference/api/tasks.md#query-parameters-1).
```

Comment:
Not super precise, but it could be enough. I would have mentioned what is acceptable to solve that error as we do line 139 with `invalid_typo_tolerance_min_word_size_for_typos`